ddsrouter_yaml/test/unittest/configuration/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ set(TEST_NAME YamlReaderConfigurationTest)
 set(TEST_SOURCES
         YamlReaderConfigurationTest.cpp
         ${PROJECT_SOURCE_DIR}/src/cpp/YamlReaderConfiguration.cpp
+        ${PROJECT_SOURCE_DIR}/src/cpp/YamlReader_configuration.cpp
```

Comment:
Change tests names. Otherwise they will not run

Reply:
I don't know what do you mean here